src/containers/NavBar.tsx

```diff
@@ -17,9 +17,21 @@ const NavBar = (props: IProps) => {
   const { nickName, setNickName } = useSettings();
 
   const list: { icon: string; label: string; type: ListType }[] = [
-    { icon: images.icons.favorite, label: "Favorites", type: "favorites" },
-    { icon: images.icons.internet, label: "Internet", type: "internet" },
-    { icon: images.icons.partner, label: "Partners", type: "partners" },
+    { 
+      icon: images.icons.favorite,
+      label: "Favorites",
+      type: "favorites"
+    },
+    {
+      icon: images.icons.internet,
+      label: "Internet",
+      type: "internet"
+    },
+    {
+      icon: images.icons.partner,
+      label: "Partners",
+      type: "partners"
+    },
```

Comment:
These formattings are done using prettier formatter, so this change is not needed and it's not correct.

Reply:
reverted